test/jdk/java/lang/System/LoggerFinder/internal/LoggerFinderLoaderTest/LoggerFinderLoaderTest.java

```diff
@@ -230,7 +230,7 @@ static LoggerFinder getLoggerFinder(Class<?> expectedClass,
                             throw new RuntimeException("Expected message not found. Error stream contained: " + warning);
                         }
                     } else if (singleton) {
-                        if (!warning.contains("java.util.ServiceConfigurationError: More than on LoggerFinder implementation")) {
+                        if (!warning.contains("java.util.ServiceConfigurationError: More than one LoggerFinder implementation")) {
```

Comment:
please add 8283049 to the `@bug` list above since the test actually verifies the fix.

Reply:
Updated, thanks!